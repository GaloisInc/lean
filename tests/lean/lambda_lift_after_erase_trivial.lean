structure box (α : Type) :=
(val : α)

set_option trace.compiler.lambda_lifting true

def f1 : box (ℕ → ℕ) :=
box.mk id
