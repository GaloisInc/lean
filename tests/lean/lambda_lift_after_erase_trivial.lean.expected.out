[compiler.lambda_lifting] 
>> f1
λ (a : ℕ), a
